### PR TITLE
New 2.4.5 getting started guide

### PR DESCRIPTION
**Pull request by ChristopherKB**
Revised Getting Started Guide and made small updates or corrections to several other pages in the documentation.

### CODE REVIEW

File: pages/getting_started/collect.rst [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+* Plain/Raw Text (TCP, UDP, AMQP, Kafka)
+
+Content packs
+=============
+
+Additional inputs may be installed via content packs. Content packs are bundles of Graylog input, extractor, stream, dashboard, and output configurations that can provide full support
+for a data source. Some content packs are shipped with Graylog by default and some are available from the website. Content packs that
+were downloaded from `the Graylog Marketplace <http://marketplace.graylog.org>`__ can be imported using the Graylog web interface.
+
+You can load and even create your own content packs from the ``System -> Content Packs`` section of your Graylog web interface.
```
Comment (jalogisch) [inline pages/getting_started/collect.rst]: we have in the complete docs ``System / Content Packs`` and not ``System -> Content Packs`` 

Just to have it the same

File: pages/getting_started/configure.rst [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+Initial Configuration
+*********************
+
+Once the application is installed, there are a few items that must be configured before Graylog may be started for the first time. Both the Graylog ``server.conf`` and Elasticsearch ``elasticsearch.yml`` files are configuration files that contain key details needed for intial configuration. 
+
+This guide will provide you with the essential settings to get Graylog up and running. There are many other important settings in these files and we encourage you to review them once you are up and running. For more details, please see :ref:`server/.conf`
+
+.. note:: If you are using the virtual appliance, please skip this section and go directly to :ref:`connect`
+
+server.conf
```
Comment (jalogisch) [inline pages/getting_started/configure.rst]: I'm not happy having the double content in the docs - But I did not know a better solution currently.

We need to remember in the future during edits of the server.conf description that parts are mirrored in this page.

File: pages/getting_started/download_install.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+Operating System Packages
+^^^^^^^^^^^^^^^^^^^^^^^^^
+Graylog may be installed on the following operating systems.
+
+* Ubuntu
+* Debian
+* RHEL/CentOS
+* SLES
+
+Most customers use package tools like DEB or RPM to install the Graylog software. Details are included in the section, :ref:`ospackages`
```
Comment (jalogisch) [inline pages/getting_started/download_install.rst]: missing `.`?

File: pages/getting_started/download_install.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+
+Configuration Management
+^^^^^^^^^^^^^^^^^^^^^^^^
+Customers who prefer to deploy graylog via configuration management tools may do so. Graylog currently supports :ref:`confmgt`
```
Comment (jalogisch) [inline pages/getting_started/download_install.rst]: missing `.`?

File: pages/getting_started/explore.rst [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+Content Packs
+"""""""""""""
+
+Content packs accelerate the set-up process for a specific data source. A content pack can include inputs/extractors, streams, dashboards, alerts, saved searches and pipeline processors. 
+
+Any program element created within Graylog may be exported as Content Packs for use on other systems. These may be kept private by the author, for use in quick deployment of new nodes internally, or may be shared with the community via the Graylog Marketplace. For example, users may create custom Inputs, Streams, Dashboards, and Alerts to support a security use case. These elements may be exported in a content pack and then imported on a newly  installed Graylog instance to save configuration time and effort. 
+
+Users may download content packs created and shared by other users via the :ref:`marketplace`. User created content packs are not supported by Graylog, but instead by their authors.
+
+.. List of Supported Elements
```
Comment (jalogisch) [inline pages/getting_started/explore.rst]: Is content here missing? Did you want to add something more? Shall we kick this?

File: pages/getting_started/web_console.rst [hunk 6]
```diff
@@ -1,11 +1,15 @@
+.. _connect:
```
Comment (jalogisch) [inline pages/getting_started/web_console.rst:1]: in future, use little longer anchors - having short is nice to write but having better names will make it easy to remember where this links to.

File: pages/installation/aws.rst [hunk 7]
```diff
@@ -1,5 +1,7 @@
+.. _AMI:
+
 *******************
-Amazon Web Services
+AWS - AMIs
 *******************
```
Comment (jalogisch) [inline pages/installation/aws.rst]: the headline markers `**` need to match the length of the headline!

**Review by jalogisch — CHANGES_REQUESTED** [review]
Chris could you please recheck everything, search for missing dots and similar.

Left some comments you might want to work on.

File: pages/getting_started/configure.rst (hunk 2 above)
Comment (ChristopherKB) [reply pages/getting_started/configure.rst]: I agree completely, but I don't have a better solution either.

**Review by jalogisch — APPROVED** [review]
LGTM

**Pull request merged**